js-exercises/telephone-check/telephoneCheck.js

```diff
@@ -0,0 +1,6 @@
+function telephoneCheck(phoneNumber) {
+  const phoneNumberRegex = /^(1\s?)?((\([0-9]{3}\))|[0-9]{3})[\s\-]?[\0-9]{3}[\s\-]?[0-9]{4}$/;
```

Comment:
giving a comment on regex is good option like this one   " will help to validate the proper us numbers  "